IaC/psmdb.cd/init.groovy.d/cloud.groovy

```diff
@@ -166,35 +231,51 @@ initMap['min-jessie-x64'] = '''
     rm -fv jre-8u152-linux-x64.tar.gz
     sudo install -o $(id -u -n) -g $(id -g -n) -d /mnt/jenkins
 '''
+initMap['min-bionic-x64'] = '''
+    set -o xtrace
+    until sudo apt-get update; do
+        sleep 1
+        echo try again
+    done
+
+    if ! mountpoint -q /mnt; then
+        DEVICE=$(ls /dev/xvdd /dev/nvme0n1 | head -1)
+        sudo apt-get -y install xfsprogs
+        sudo mkfs.xfs ${DEVICE}
+        sudo mount ${DEVICE} /mnt
+    fi
+
+    sudo apt-get -y install openjdk-8-jre-headless git
+    sudo install -o $(id -u -n) -g $(id -g -n) -d /mnt/jenkins
+'''
 initMap['min-stretch-x64'] = initMap['min-artful-x64']
-initMap['min-trusty-x64'] = initMap['min-jessie-x64']
-initMap['min-xenial-x64'] = initMap['min-artful-x64']
-initMap['min-cosmic-x64'] = initMap['min-artful-x64']
-initMap['min-bionic-x64'] = initMap['min-artful-x64']
-initMap['psmdb'] = initMap['min-xenial-x64']
-initMap['psmdb-bionic'] = initMap['min-xenial-x64']
+initMap['min-trusty-x64']  = initMap['min-jessie-x64']
+initMap['min-xenial-x64']  = initMap['min-artful-x64']
+initMap['min-cosmic-x64']  = initMap['min-artful-x64']
+initMap['psmdb']           = initMap['min-xenial-x64']
+initMap['psmdb-bionic']    = initMap['min-bionic-x64']
 
 capMap = [:]
 capMap['c4.xlarge'] = '60'
 capMap['m4.xlarge'] = '60'
-capMap['m4.2xlarge'] = '10'
+capMap['m5.2xlarge'] = '10'
 
 typeMap = [:]
-typeMap['micro-amazon'] = 't2.small'
-typeMap['docker'] = 'c4.xlarge'
-typeMap['docker-32gb'] = 'm4.2xlarge'
-typeMap['min-centos-7-x64'] = typeMap['docker-32gb']
-typeMap['fips-centos-7-x64'] = typeMap['min-centos-7-x64']
-typeMap['min-artful-x64'] = typeMap['min-centos-7-x64']
-typeMap['min-centos-6-x64'] = typeMap['docker-32gb']
-typeMap['min-jessie-x64'] = typeMap['docker-32gb']
-typeMap['min-stretch-x64'] = typeMap['docker-32gb']
-typeMap['min-trusty-x64'] = typeMap['min-centos-7-x64']
-typeMap['min-xenial-x64'] = typeMap['min-centos-7-x64']
-typeMap['min-cosmic-x64'] = typeMap['docker-32gb']
-typeMap['min-bionic-x64'] = typeMap['docker-32gb']
-typeMap['psmdb'] = typeMap['docker-32gb']
-typeMap['psmdb-bionic'] = typeMap['docker-32gb']
+typeMap['micro-amazon']      = 't2.small'
+typeMap['docker']            = 'c4.xlarge'
+typeMap['docker-32gb']       = 'm5.2xlarge'
+typeMap['min-centos-7-x64']  = typeMap['docker-32gb']
+typeMap['fips-centos-7-x64'] = typeMap['docker-32gb']
+typeMap['min-artful-x64']    = typeMap['docker-32gb']
+typeMap['min-centos-6-x64']  = typeMap['docker']
```

Comment:
why not 32gb?

Reply:
We can not use m5 instance because aws does not allow it via ec2 plugin for centos 6, so for centos 6 I am using old m4 with 32gb.